src/test/resources/fixtures/hmh-ri/caliperEventBasicCreated.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "type": "Event",
+  "actor": {
+    "id": "https://url/to/lexileService",
+    "type": "SoftwareApplication",
+    "version": "v1"
+  },
+  "action": "Created",
+  "object": {
```

Comment:
The Created action isn't really being performed on the Attempt, the object it being performed on is the DigitalResourceCollection. So I'd expect what you have in the generated property be used for object and provide no generated property. I think it is fine though to reference the Attempt which led to this resource being created in the referrer property.
The value of mediaType has to be a valid media type, like application/pdf, text/html etc.

Reply:
Thanks Padraig, a couple of things...

DigitalResourceCollection is the list of recommended books... I'm thinking that this CreatedEvent is a list of these recommendations were created by the score generated from the students specific assessment.
Yes, we can take out the attempt but I think it should be replaced with a reference the specific assessment itself i.e. the values I have here https://github.com/hmhco/caliper-common-fixtures/pull/3/files#diff-804b2bc03ee3fdd1d976f09fc3e1bbfbR20 Thoughts?